src/vs/editor/common/languages.ts

```diff
@@ -1206,6 +1206,16 @@ export function isLocationLink(thing: any): thing is LocationLink {
 		&& (Range.isIRange((thing as LocationLink).originSelectionRange) || Range.isIRange((thing as LocationLink).targetSelectionRange));
 }
 
+/**
+ * @internal
+ */
+export function isLocation(thing: any): thing is Location {
+	return thing
+		&& URI.isUri((thing as Location).uri)
+		&& Range.isIRange((thing as Location).range);
+}
```

Comment:
Btw. I'd expect that this function does not crash on untrusted values.
However, it would crash for `isLocation(null)`!

Reply:
`null` is checked on line 1213